PSvCloud/PSvCloud.psd1

```diff
@@ -69,7 +69,7 @@ PowerShellVersion = '4.0'
 # NestedModules = @()
 
 # Functions to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no functions to export.
-FunctionsToExport = @('Get-CIEdge','Get-CIEdgeView','Get-CIEdgeXML')
+FunctionsToExport = @('Get-CIEdge','Get-CIEdgeView','Get-CIEdgeXML','Get-CIEdgeSecurityCheck')

```

Comment:
This is automatically updated as part of the PSake Build task, so you could leave this next time if you wanted: https://github.com/adamrushuk/PSvCloud/blob/master/build.psake.ps1#L97